<?xml version="1.0" encoding="UTF-8"?><testsuite errors="1" failures="0" name="Gestionar cita mÃ©dica" skipped="0" tests="1" time="14.87" timestamp="2019-02-04 04:26:13"><testcase name="Realizar el Registro de un Paciente"><error message="Arity mismatch: Step Definition 'co.com.retotres.screenplay.stepdefinitions.RegistrarPacienteStepDefinitions.elVerificaEnPantallaQueElPacienteEsCreadoConElMensajeDatosGuardadosCorrectamente(String) in file:/C:/Users/cochoar/Desktop/Proyectos/RetoTres/co.com.retotres.screenplay/bin/test/' with pattern [^el verifica en pantalla que el paciente es creado con el mensaje Datos guardados correctamente\.$] is declared with 1 parameters. However, the gherkin step has 0 arguments []. &#10;Step text: el verifica en pantalla que el paciente es creado con el mensaje Datos guardados correctamente." type="cucumber.runtime.CucumberException">Arity mismatch: Step Definition 'co.com.retotres.screenplay.stepdefinitions.RegistrarPacienteStepDefinitions.elVerificaEnPantallaQueElPacienteEsCreadoConElMensajeDatosGuardadosCorrectamente(String) in file:/C:/Users/cochoar/Desktop/Proyectos/RetoTres/co.com.retotres.screenplay/bin/test/' with pattern [^el verifica en pantalla que el paciente es creado con el mensaje Datos guardados correctamente\.$] is declared with 1 parameters. However, the gherkin step has 0 arguments []. 
Step text: el verifica en pantalla que el paciente es creado con el mensaje Datos guardados correctamente.</error><system-err>Arity mismatch: Step Definition 'co.com.retotres.screenplay.stepdefinitions.RegistrarPacienteStepDefinitions.elVerificaEnPantallaQueElPacienteEsCreadoConElMensajeDatosGuardadosCorrectamente(String) in file:/C:/Users/cochoar/Desktop/Proyectos/RetoTres/co.com.retotres.screenplay/bin/test/' with pattern [^el verifica en pantalla que el paciente es creado con el mensaje Datos guardados correctamente\.$] is declared with 1 parameters. However, the gherkin step has 0 arguments []. 
Step text: el verifica en pantalla que el paciente es creado con el mensaje Datos guardados correctamente.&#13;
cucumber.runner.Runner.runPickle(Runner.java:80)&#13;
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:538)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:760)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:460)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:206)&#13;
</system-err></testcase></testsuite>